config/development.json

```diff
@@ -46,5 +46,5 @@
 		"SECURE": false,
 		"EXPIRES_SECONDS": 2592000000
 	},
-	"HYDRA_URI": "http://localhost:9001"
+	"HYDRA_URI": "http://hydra.localhost:9001"
```

Comment:
I have never seen this before... does this work for default local hydra deployments?

Reply:
this is an advanced config for developing with a running hydra and nextcloud instance at once. removed for commit